kotlin/internal-testing-utils/src/test/java/com/squareup/workflow/internal/util/UncaughtExceptionGuardTest.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Square Inc.
+ * Copyright 2019 Square Inc.
```

Comment:
Updating copyright dates is weird. There is still a lot of code that was written two years ago, a newer date is a weaker claim. (IANAL.)